infra/helper.py

```diff
@@ -95,8 +95,8 @@ def main():
 
   check_build_parser = subparsers.add_parser(
       'check_build', help='Checks that fuzzers execute without errors.')
-  _add_engine_args(check_build_parser)
-  _add_sanitizer_args(check_build_parser)
+  _add_engine_args(check_build_parser, ['libfuzzer', 'afl'])
```

Comment:
honggfuzz :) ?

Reply:
Intentional omission. I don't think bad_build_check is intended to work with honggfuzz

Reply:
sgtm